README.md

```diff
@@ -15,12 +15,20 @@ The builder is primarily a git server that responds to `git push`es by executing
 
 1. Calls `git archive` to produce a tarball (i.e. a `.tar.gz` file) on the local file system
 2. Saves the tarball to centralized object storage according to the following rules:
-	- If the `DEIS_OUTSIDE_STORAGE` environment variable exists, saves to the [S3 API][s3-api-ref] compatible server at `https://$DEIS_OUTSIDE_STORAGE`
-  - Otherwise, if the `DEIS_MINIO_SERVICE_HOST` and `DEIS_MINIO_SERVICE_PORT` environment variables exist (these are standard [Kubernetes service discovery environment variables](http://kubernetes.io/docs/user-guide/services/#environment-variables)), saves to the [S3 API][s3-api-ref] compatible server at `http://$DEIS_MINIO_SERVICE_HOST:$DEIS_MINIO_SERVICE_HOST`
+	- If the `BUILDER_STORAGE` environment variable is other than `minio`, attempts to create the appropriate storage driver and saves using this driver.
```

Comment:
Definitely looking for help to clarify/amend this section -- did my best after reading through code I'm new to but sure I haven't grasped the complete picture.  ping @bacongobbler & @mboersma